source/includes/getting-started/_concepts.html.md

```diff
@@ -1,61 +1,67 @@
-# Architecture
+# Background FAQ
 
-There are some things you might want to know before you get started. In this section, we're going to walk you through the Bonsai Platform components and some of the basic Bonsai concepts. Let's jump right in!
+[//]: # (If any commented statements become false, change prior text)
+
+There are some things you might want to know before you get started. In this section, we're going to walk you through the Bonsai Platform components and some of the basic Bonsai concepts that are frequently asked questions. Let's jump right in!
 
 ## What are the components of the Bonsai Platform?
 
-The image below displays all of the components of Bonsai's platform. It enables  developers to more efficiently Build, Teach and Use intelligence models:
+This image displays all of the components of Bonsai's Platform. The Platform enables developers to more efficiently build, teach and use intelligence models:
 
 ![The Bonsai Platform][1]
 
-## Build
+###### Build
 
-* **CLI:** the Bonsai Command Line Interface (CLI) tool.
-* **brains.bons.ai:** The address for Bonsai's web interface; this is the browser based tool that includes your BRAIN Dashboard and BRAIN Details page.
-* **Simulators:** are a training source. They imitate a real world situation in a virtual environment. Simulators are often a complex, interactive environment. They usually have state, which is a representation of the world inside the virtual environment.
+* **Bonsai Command Line Interface (CLI):** is a command line tool that enables you to configure and control the Bonsai Artificial Intelligence (AI) Engine.
+* **beta.bons.ai:** The address for Bonsai's beta web interface; this is the browser based tool that includes your *BRAIN Dashboard* and *BRAIN Details* page.
+* **Simulators:** are currently the only training source implemented. Simulators are often a complex, interactive virtual environment. They usually have state, which is a representation of the world inside the virtual environment.
```

Comment:
The word "training source" should be defined. Most tools currently only support batch data so the word "training source" will be new to many users.  A training source is any source of data used to teach or train an AI. It includes batch data like a large spreadsheet or a set of labels and images and synthetic data like the output from a simulation.